RELEASE_NOTES.md

```diff
@@ -7,8 +7,8 @@ Example Note:
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
 
-# azure-chef-extension 1210.12.100.1000 release notes:
-In this release we have added functionality not to uninstall chef-client even if the extension gets uninstalled.
+# azure-chef-extension 1210.12.101.1000 for Windows and 1210.12.101.1000 for Ubuntu release notes:
+In this release we have added extension support for Debian platform, added functionality to determine whether to uninstall chef-client or not using flag uninstall_chef_client and installing chef from Omnitruck for Ubuntu CentOS and Debian

```

Comment:
Here's a suggestion. This could be modified as:
`In this release we have added extension support for Debian platform. We have also added an option uninstall_chef_client which determines whether to uninstall chef-client or not during Extension Update and Uninstall. Now we are installing chef from Omnitruck for Ubuntu, CentOS and Debian.`